packages/wrangler/src/dev/miniflare.ts

```diff
@@ -18,10 +18,10 @@ import {
 } from "../ai/fetcher";
 import { readConfig } from "../config";
 import { ModuleTypeToRuleType } from "../deployment-bundle/module-collection";
-import { stripExperimentalPrefixes } from "../deployment-bundle/node-compat";
 import { withSourceURLs } from "../deployment-bundle/source-url";
 import { UserError } from "../errors";
 import { logger } from "../logger";
+import { stripExperimentalPrefixes } from "../node-compat";
```

Comment:
These changes are because the node_compat helpers have been moved to a location where it makes more sense (outside the `deployment-bundle` folder).